applications/search/src/api/datasets.js

```diff
@@ -16,3 +18,16 @@ export const getDatasetByURI = uri =>
     .get(`${datasetsUrlBase}/byuri?uri=${uri}`)
     .then(response => response.data)
     .catch(e => console.log(JSON.stringify(e))); // eslint-disable-line no-console
+
+export function extractDatasets(searchResponse) {
+  const hits = get(searchResponse, 'hits.hits', []);
+  return hits.map(hit => hit._source);
```

Comment:
The two lines could be done as one operation

Reply:
fikset